storage/cloudspanner/map_storage_test.go

```diff
@@ -42,12 +58,68 @@ func GetTestDB(ctx context.Context, t *testing.T) *spanner.Client {
 	return client
 }
 
+// Unique per test binary invocation
+var timestamp = time.Now().UTC().Format("jan-02-15-04-05")
```

Comment:
```
var (
  ...
)
```

Also, should these be inside the `uniqueDBName` func?

Reply:
some, yes